test/cat.jl

```diff
@@ -62,6 +62,15 @@ module TestCat
         @test df[2] == collect(1:10)
     end
 
+    @testset "hcat ::Vectors" begin
+        df = DataFrame()
+        df2 = hcat(DataFrames.CategoricalVector{Union{Int, Missing}}(1:10), df)
```

Comment:
Why the `DataFrames.` prefix?

Also test that the result contains a single column, and check its name, that doesn't hurt.

Reply:
prefix - removed.
additional checks added